cpp/src/prims/detail/extract_transform_v_frontier_e.cuh

```diff
@@ -0,0 +1,1039 @@
+/*
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION.
```

Comment:
Is this new (should be just 2022) or moved from somewhere?

Reply:
Copied from transfrom_reduce_v_frontier_outgoing_e_by_dst.cuh